DakeAndDevileCorps/Screens/MapHome/VC/MainMapViewController.swift

```diff
@@ -51,3 +107,24 @@ extension MainMapViewController: SearchBarDelegate {
         present(nextViewController, animated: true)
     }
 }
+
+// MARK: - CategoryCollectionViewDelegate
+extension MainMapViewController: CategoryCollectionViewDelegate {
+    func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
+        mapView.removeAnnotations(mapView.annotations)
+        let categoryName = categoryView.categoryList[indexPath.row]
+
+        for shop in shops {
+            guard let shop = shop as? ShopInfo else { return }
+            if shop.sellingProductsCategory.contains(categoryName) {
+                mapView.addAnnotation(shop)
+            }
+        }
```

Comment:
p5;

```suggestion
        shops.forEach { shop in
            guard let shop = shop as? ShopInfo else { return }
            if shop.sellingProductsCategory.contains(categoryName) {
                mapView.addAnnotation(shop)
            }
        }
```
이렇게 사용할 수도 있을 듯 합니다..!

Reply:
좋은 것 같아요. 반영했습니다! [9db9673](https://github.com/DeveloperAcademy-POSTECH/MC3-Team4-DakeAndDevileCorps/pull/69/commits/9db9673c1c9e2225e68b7374632619e6626a4056)